vendors.json

```
@@ -94,6 +94,11 @@
     "env": ["JENKINS_URL", "BUILD_ID"],
     "pr": { "any": ["ghprbPullId", "CHANGE_ID"] }
   },
+  {
+    "name": "ZEIT Now",
+    "constant": "ZEIT_NOW",
+    "env": "NOW_BUILDER_ANNOTATE"
```

We've actually changed this variable to be just `NOW_BUILDER`.

```suggestion
    "env": "NOW_BUILDER"
```